feature-configs/cn-ran-overlays/README.md

```diff
@@ -0,0 +1,25 @@
+# cn-ran-overlays
+
+This folder contains an example configurations for the RAN sites.
+Current example is focused on DU. CU profile can be built by
+- Replacing PTP by NTP (Work in progress)
+- Using stock kernel instead of RT kernel
+- Skipping FEC (Work in progress)
+
+
+## Overview
+
+The [`all-in-one`](all-in-one) directory contains the Kustomize profile for deployment of DU integration features, namely:
+- SCTP MachineConfig patch
+- Performance addon operator and DU performance profile
+- PTP operator and slave profile
+- SR-IOV operator and associated profiles
+
+## Deployment
+
+1. Make sure your nodes are labeled as required and MCP is created. If your cluster is brand new, this can be done by cnf-features-deploy/hack/setup-test-cluster.sh
```

Comment:
I'd speak about pre-requisites here.
What we need is a machine config pool and a set of nodes included in that pool.

The `make setup-test-cluster` command will create the machine config pool referenced in the example configuration named "worker-cnf" and will label the nodes accordingly.

Reply:
Btw, plain worker pool is ok as well. There is no need to create additional one. Also there must be at least one unique label on the MCP, because the performance profile can be configured to point to it (instead of trying to be smart).